.github/workflows/autotools.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: autoreconf
+      run: autoreconf -if
```

Comment:
Please remove this. It is our policy to manually regenerate `configure` and commit it to the repo (yes, we forgot this last week).